backend/src/schema/resolvers/posts.js

```diff
@@ -92,15 +94,21 @@ export default {
       const createPostVariables = { userId: context.user.id, categoryIds, params }
 
       const session = context.driver.session()
-      const transactionRes = await session.run(createPostCypher, createPostVariables)
-
-      const [post] = transactionRes.records.map(record => {
-        return record.get('post')
-      })
-
-      session.close()
+      try {
+        const transactionRes = await session.run(createPostCypher, createPostVariables)
+        const posts = transactionRes.records.map(record => {
+          return record.get('post').properties
+        })
+        post = posts[0]
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-09T06:57:25Z" title="Monday, September 9th 2019, 8:57:25 am +02:00">Sep 9, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -92,15 +94,22 @@ export default {
       const createPostVariables = { userId: context.user.id, categoryIds, params }
 
       const session = context.driver.session()
-      const transactionRes = await session.run(createPostCypher, createPostVariables)
-
-      const [post] = transactionRes.records.map(record => {
-        return record.get('post')
-      })
-
-      session.close()
+      try {
+        const transactionRes = await session.run(createPostCypher, createPostVariables)
+        const posts = transactionRes.records.map(record => {
+          return record.get('post').properties
+        })
+        debugger
```


I can't think of any reason why you'd want to leave this in here, but instead of removing it, I'll just point it out.